acceptance/suites/pre_suite/foss/30_install_dev_repos.rb

```diff
@@ -1,27 +1,19 @@
-install_opts = options.merge( { :dev_builds_repos => ["PC1"] })
-repo_config_dir = 'tmp/repo_configs'
+if test_config[:puppetserver_install_type] == :package
+  package_build_version = ENV['PACKAGE_BUILD_VERSION']
+  if package_build_version.nil?
+    abort("Environment variable PACKAGE_BUILD_VERSION required for package installs!")
+  end
 
-case test_config[:puppetserver_install_type]
-when :package
-  step "Setup Puppet Server repositories." do
-    package_build_version = ENV['PACKAGE_BUILD_VERSION']
-    if package_build_version
-      install_puppetlabs_dev_repo master, 'puppetserver', package_build_version,
-                                  repo_config_dir, install_opts
-    else
-      abort("Environment variable PACKAGE_BUILD_VERSION required for package installs!")
-    end
+  step "Setup Puppet Server dev repository on the master." do
+    install_opts = options.merge({ :dev_builds_repos => ["PC1"]})
+    install_puppetlabs_dev_repo(master, 'puppetserver', package_build_version, nil, install_opts)
   end
 end
 
-puppet_build_version = test_config[:puppet_build_version]
-if puppet_build_version
-  confine_block :except, :platform => ['windows'] do
-    step "Setup Puppet Labs Dev Repositories." do
-      hosts.each do |host|
-        install_puppetlabs_dev_repo host, 'puppet-agent', puppet_build_version,
-                                    repo_config_dir, install_opts
-      end
+confine_block :except, :platform => ['windows'] do
+  step "Setup Puppet Labs Dev Repositories." do
+    hosts.each do |host|
+      install_puppet_agent_dev_repo_on(host, {:puppet_agent_version => test_config[:puppet_build_version]})
```

Comment:
Yeah, it's probably fine this way.  I honestly can't think of a good reason not to always have a version there - even if in the future we need to do something different to derive the "latest good version" from some other source rather than an explicit pin like we do today.